tests/e2e/support/api/keycloak/user.ts

```diff
@@ -43,19 +43,27 @@ export const createUser = async ({ user, admin }: { user: User; admin: User }):
   checkResponseStatus(creationRes, 'Failed while creating user')
 
   // created user id
-  const uuid = getUserIdFromResponse(creationRes)
+  const keycloakUuid = getUserIdFromResponse(creationRes)
 
   // assign realmRoles to user
   const defaultNewUserRole = 'User'
-  const roleRes = await assignRole({ admin, uuid, role: defaultNewUserRole })
+  const roleRes = await assignRole({ admin, uuid: keycloakUuid, role: defaultNewUserRole })
   checkResponseStatus(roleRes, 'Failed while assigning roles to user')
 
   const usersEnvironment = new UsersEnvironment()
-  usersEnvironment.storeCreatedUser({ user: { ...user, uuid, role: defaultNewUserRole } })
+  //stored keycloak user information on storage
+  usersEnvironment.storeCreatedKeycloakUser({
+    user: { ...user, uuid: keycloakUuid, role: defaultNewUserRole }
+  })
 
-  // initialize user
+  // initialize user on Ocis web
+  // need login to have the user created by keycloak appear on Ocis web
```

Comment:
```suggestion
  // login to initialize the user in oCIS
```

Reply:
doesnot this comment is similar to upper comment
```suggestion
  // initialize user on Ocis web
  // login to initialize the user in oCIS
```

Reply:
keep only one comment. should be enough

Reply:
Okay